plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -564,7 +571,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                   fc_dlog(_trx_failed_trace_log, "[TRX_TRACE] Speculative execution is REJECTING tx: ${txid}, auth: ${a} : ${why} ",
                           ("txid", trx->id())
                           ("a", trx->packed_trx()->get_transaction().first_authorizer())
-                          ("why", except_ptr->what()));
+                          ("why", details));
```

Comment:
The `push_transaction` in `process_unapplied_trxs` should also be updated to log this information. Or just wait until https://github.com/eosnetworkfoundation/mandel/issues/297 which should fix that issue.

Reply:
It's better to leave changes for process_unapplied_trxs to #297. I have added a reminder to that issue.